server/pkg/provision/provision.go

```diff
@@ -42,6 +42,7 @@ func Run(ctx context.Context, log *zap.Logger, s store.Storer, provisionOpt opti
 		func() error { return authAddExternals(ctx, log.Named("auth.externals"), s) },
 		func() error { return oidcAutoDiscovery(ctx, log.Named("auth.oidc-auto-discovery"), s, authOpt) },
 		func() error { return defaultAuthClient(ctx, log.Named("auth.clients"), s, authOpt) },
+		func() error { return addAuthSuperUsers(ctx, log.Named("auth.super-users"), s) },
```

Comment:
can you use authOpt as the last argument and lose the `val = options.AuthOpt()...` in the body of the function?